### ISSUE TITLE
[README]Url for "User Model" and "Database Connection"  are invalid

### ISSUE THREAD

**Issue by bobzhen** [trenpixster/addict#106]
Hello,

I found that these two url does not exist anymore on github:

```
https://github.com/elixir-ecto/ecto/blob/master/examples/simple/lib/simple.ex#L18
https://github.com/elixir-ecto/ecto/blob/master/examples/simple/lib/simple.ex#L12
```

which you can find them on README.md, first paragraph of Requirements section

```
"Addict is dependent on an ecto User Model and a Database connection interface."
```

Could you please update it? 

I would like to help but I have no idea about the original source.

Thanks!

**Comment by schmijos** [comment]
Trying to fix it with https://github.com/trenpixster/addict/pull/113

**Comment by trenpixster** [comment]
Closed via #113
